react-client/src/Components/App.js

```diff
@@ -112,12 +107,19 @@ function App() {
   };
 
   const handleRemoveDialogBox = (idx) => {
-
     const newDomInfo = domInfo.filter((x, currentIdx) => currentIdx !== idx);
 
     setDomInfo(newDomInfo);
-  }
-  // const resetdom = () => setshowSwitch(true);
+  };
+
+  const handleSeemore = () => {
+    debugger
+   try {
+     setdisplayArray(domInfo.childcount);
```

Comment:
Why use `childcount` on this? This is redundant. Remember that you have `children` array that you can fetch use to get the # of child.

Reply:
Remove "debugger" once you're done with the task

Reply:
noted already done and modified